### PR TITLE
this was successfully run 11/09

### PR DESCRIPTION
**Pull request by guinneak198**
debugging errors that popped up when I tried to run

### CODE REVIEW

File: examples/combined_ODNP.py [hunk 1]
```diff
@@ -244,6 +244,8 @@
             ret_data=DNP_data,
         )
         DNP_thermal_done = time.time()
+        if j == 0:
+            time_axis_coords = DNP_data.getaxis("indirect")
```
Comment (guinneak198) [inline examples/combined_ODNP.py:248]: time_axis_coords needs to be defined at some point

File: examples/combined_ODNP.py [hunk 2]
```diff
@@ -311,7 +313,7 @@
             final_log.append(
                 "if I got this far, that probably worked -- be sure to move/rename temp_ODNP.h5 to the correct name!!"
     logger.info("\n*** FILE SAVED IN TARGET DIRECTORY ***\n")
-    logger.debug(strm("Name of saved data", echo_data.name()))
+    logger.debug(strm("Name of saved data", DNP_data.name()))
```
Comment (guinneak198) [inline examples/combined_ODNP.py:316]: no such thing as echo_data

File: examples/combined_ODNP.py [hunk 3]
```diff
@@ -378,14 +380,10 @@
...
         print(("Name of saved data", vd_data.name()))
-    final_frq = p.dip_lock(
-        config_dict["uw_dip_center_GHz"] - config_dict["uw_dip_width_GHz"] / 2,
-        config_dict["uw_dip_center_GHz"] + config_dict["uw_dip_width_GHz"] / 2,
-    )
     this_log = p.stop_log()
 # }}}
 config_dict.write()
-with h5py.File(os.path.join(target_directory, filename), "a") as f:
+with h5py.File(os.path.normpath(os.path.join(target_directory, filename)), "a") as f:
```
Comment (guinneak198) [inline examples/combined_ODNP.py:386]: using this the log saved successfullly in the right h5 file

File: examples/combined_ODNP.py [hunk 4]
```diff
@@ -378,14 +380,10 @@
         vd_data.hdf5_write(filename, directory=target_directory)
         print("\n*** FILE SAVED IN TARGET DIRECTORY ***\n")
         print(("Name of saved data", vd_data.name()))
-    final_frq = p.dip_lock(
-        config_dict["uw_dip_center_GHz"] - config_dict["uw_dip_width_GHz"] / 2,
-        config_dict["uw_dip_center_GHz"] + config_dict["uw_dip_width_GHz"] / 2,
-    )
```
Comment (guinneak198) [inline examples/combined_ODNP.py:384]: not needed!

File: examples/combined_ODNP.py [hunk 5]
```diff
@@ -65,7 +65,7 @@
 dB_settings = Ep_spacing_from_phalf(
     est_phalf = config_dict['guessed_phalf'],
     max_power = config_dict["max_power"], 
-    config_dict["power_steps"] + 1, 
+    p_steps = config_dict["power_steps"] + 1,
```
Comment (jmfranck) [inline examples/combined_ODNP.py:68]: why are you feeding +1 here? Why not just config_dict['power_steps']?

**Review by jmfranck — CHANGES_REQUESTED** [review]
one question

File: examples/combined_ODNP.py (hunk 5 above)
Comment (guinneak198) [reply examples/combined_ODNP.py:68]: this was pulled in from previous runs (prior to when we had thermals in a separate loop to accomodate a power axis long enough for including the thermals. - but you point out that this is no longer needed

**Pull request merged**